app/Controllers/Http/AuthController.ts

```diff
@@ -0,0 +1,42 @@
+import type { HttpContextContract } from '@ioc:Adonis/Core/HttpContext'
+import User from 'App/Models/User'
+import Profile from 'App/Models/Profile'
+import RegisterValidator from 'App/validators/RegisterValidator'
+
+export default class AuthController {
+  public async register({ request, response, auth }: HttpContextContract) {
+    const { email, password, mobile } = await request.validate(RegisterValidator)
+
+    // create user
+    const user = await User.create({ email, password })
+
+    // create profile
+    await Profile.create({
+      mobile: mobile,
+      user_id: user.id,
```

Comment:
pass name, dob and gender also while creating profile

Reply:
fixed